rebuild-ui is broken

```
$ make rebuild-ui
::group::rebuild-ui logs
bash ./scripts/rebuild-ui.sh
+ cd jaeger-ui
+ git fetch --all --unshallow --tags
fatal: --unshallow on a complete repository does not make sense
make: *** [rebuild-ui] Error 128
```

cc @andreasgerstmayr

--- a/scripts/rebuild-ui.sh
+++ b/scripts/rebuild-ui.sh
@@ -7,7 +7,11 @@ set -euxf -o pipefail
 
 cd jaeger-ui
 
-git fetch --all --unshallow --tags
+git fetch --all --tags
+is_shallow_repo=$(git rev-parse --is-shallow-repository)
+if [[ "$is_shallow_repo" == "true" ]]; then
+    git fetch --unshallow
+fi
 git log --oneline --decorate=full -n 10 | cat
 
 last_tag=$(git describe --tags --dirty 2>/dev/null)

--- a/scripts/rebuild-ui.sh
+++ b/scripts/rebuild-ui.sh
@@ -7,11 +7,10 @@ set -euxf -o pipefail
 
 cd jaeger-ui
 
-git fetch --all --tags
-is_shallow_repo=$(git rev-parse --is-shallow-repository)
-if [[ "$is_shallow_repo" == "true" ]]; then
+if [[ "$(git rev-parse --is-shallow-repository)" == "true" ]]; then
     git fetch --unshallow
 fi
+git fetch --all --tags
 git log --oneline --decorate=full -n 10 | cat
 
 last_tag=$(git describe --tags --dirty 2>/dev/null)

